app/models/scc_account.rb

```diff
@@ -40,8 +67,103 @@ def sync_status
     end
   end
 
+  def use_recurring_logic?
+    self.interval != NEVER
+  end
+
+  def save_with_logic!
+    self.task_group ||= SccAccountSyncPlanTaskGroup.create!
+
+    associate_recurring_logic if self.valid?
+
+    self.save!
+    start_recurring_logic if self.use_recurring_logic?
+
+    true
+  end
+
+  def update_attributes_with_logics!(params)
```

Comment:
The other one is named `*_with_logic?`. Which one is better? Or should it be `*_with_recurring_logic?`?
Do we not always update the logic when we modify / create those accounts? Should we overwrite the existing `update*`, `save*` methods?

Reply:
Changed the name to `_logic!`

Not sure if we should overwrite `update*` and `save*` the implementation was inspired by the one for the Katello SyncPlans.

Reply:
Well then, i'll ACK this.